hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -799,6 +801,34 @@ public OmKeyInfo lookupKey(OmKeyArgs args) throws IOException {
     return OmKeyInfo.getFromProtobuf(resp.getKeyInfo());
   }
 
+  @Override
+  public KeyInfoWithVolumeContext getKeyInfo(OmKeyArgs args,
```

Comment:
Would `lookupKeyWithVolumeContext` be more consistent with existing `lookupKey`?  (I assume `lookupKey` has to be retained for compatibility.)

Reply:
It's fair. Yet I think we're trying to deprecate the `lookup` APi and replace it with an optimized API. The name `get` sounds more appropriate as this API takes a key identifier (vol/bucket/keyname) and returns KeyInfo and it does zero lookup/search. 
How do you think @kerneltime?

Reply:
Lookup is a very specific API in the context of filesystems, and this API is nothing like it. It is more akin to get attributes of a file.